CONTRIBUTING.md

```diff
@@ -69,11 +69,32 @@ take several minutes.
 ## Pull Request Process
 
 1. Ensure your code follows the style guide and is portable; remove any references to local paths or files.
-2. Document the header files and public functions with doxygen comments, and update any relevant README.md
-   or documentation files with details of changes to the interface.
-3. Open a pull request into the `develop` branch. Write a meaningful title and description for the PR to make it
+2. Check that the new changes are appropriately covered by unit tests.
+3. Update the [changelog](CHANGELOG.md) with your feature / fix / improvement in the "Upcoming changes" section.
+   Ensure any other relevant documentation is also updated (including READMEs or doxygen headers).
```

Comment:
I find it unfortunate that you went from precise instructions (header files and public functions) to just saying 'other relevant' documentation. Any way we could combine these two formulations?

Reply:
Ok, I guess I could just leave the original as it was and add the new information separately:
```
X.   Document the header files and public functions with doxygen comments, and update any relevant README.md
    or documentation files with details of changes to the interface.
X+1. Update the [changelog](CHANGELOG.md) with your feature / fix / improvement in the "Upcoming changes" section.

```